applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py

```diff
@@ -260,37 +260,68 @@ def GetMinimumBufferSize(self):
         return self.min_buffer_size
 
     def AddDofs(self):
-        settings = self.main_model_part.ProcessInfo[KratosMultiphysics.CONVECTION_DIFFUSION_SETTINGS]
+        # settings = self.main_model_part.ProcessInfo[KratosMultiphysics.CONVECTION_DIFFUSION_SETTINGS]
+
```

Comment:
A lot of commented code

Reply:
WIP. It is to have something to put hands on.